feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.html

```diff
@@ -0,0 +1,111 @@
+<ng-container *ngIf="ovGroups$ | async as ovGroups; else ghostMenu">
+  <div class="cx-sticky-menu">
+    <div class="cx-menu-bar">
+      <button role="tab" class="active">
+        {{ 'configurator.overviewMenu.title' | cxTranslate }}
+      </button>
+    </div>
+    <ng-container
+      *ngTemplateOutlet="
+        groups;
+        context: {
+          overviewGroups: ovGroups,
+          level: 1,
+          idPrefix: ''
+        }
+      "
+    ></ng-container>
+  </div>
+</ng-container>
+
+<ng-template
+  #groups
+  let-overviewGroups="overviewGroups"
+  let-level="level"
+  let-idPrefix="idPrefix"
+>
+  <ng-container *ngFor="let group of overviewGroups">
+    <div [ngClass]="getGroupLevelStyleClasses(level)">
+      <span class="cx-visually-hidden">
+        {{
+          'configurator.a11y.group'
+            | cxTranslate
+              : {
+                  group: group.groupDescription
+                }
+        }}
+      </span>
+      <button
+        class="cx-menu-item"
+        role="tab"
+        (click)="navigateToGroup(idPrefix, group.id)"
+      >
+        <span title="{{ group.groupDescription }}">
```

Comment:
Is there a reason for adding a title attribute? It would lead to multiple times of reading the same info in screen readers.